docs/Telemetry.md

```diff
@@ -0,0 +1,18 @@
+# Telemetry
+
+## Metrics
+
+If [enabled][metrics feature], Statsd compatible metrics are available to use:
+
+| Name                                           | Type    |
+| ---------------------------------------------- | ------- |
+| zonemaster.rpcapi.requests.\<METHOD>.\<STATUS> | Counter |
+| zonemaster.testagent.tests_started             | Counter |
+| zonemaster.testagent.tests_completed           | Counter |
+| zonemaster.testagent.tests_died                | Counter |
+| zonemaster.testagent.tests_duration_seconds    | Timing  |
+| zonemaster.testagent.running_processes         | Gauge   |
+| zonemaster.testagent.maximum_processes         | Gauge   |
+
+
+[metrics feature]: Installation.md#d1-metrics
```

Comment:
Is it possible to add some information on how to use it, e.g. from the description of this PR?

Reply:
I would prefer not to. Statsd metrics can be integrated to a large number of backend (e.g. opensource timeseries database like prometheus using statsd exporter, influxdb using telegraf or graphite or proprietary hosted services like datadog or aws cloudwatch) giving some information on how to integrate will only cover a specific use case which may not corresponding to what the user need. What I can do is link back to the statsd repository on github which provide basic instruction for the Graphite integration.

Reply:
Just because there are many solutions does not mean that it is worthless to include some example to make it possible to quickly test something. I have no opinion or experience of statsd, but I suggest that something like the text in your comment https://github.com/zonemaster/zonemaster-backend/pull/844#issuecomment-919092486 should be included.

One aspect of it is that the Zonemaster project needs some documentation so that we can use and test the feature.

Please update the file or give some convincing arguments why it would be bad to document, and how you see code testing to be done in the future if we have no documentation.